pipeline/pipeline.go

```diff
@@ -450,6 +450,12 @@ func (p *Pipeline) streamEvent(event *Event) uint64 {
 		if node != nil {
 			event.streamName = StreamName(node.AsString())
 		}
+
```

Comment:
https://github.com/ozontech/file.d/pull/182/files#diff-78caab79060ca671036a14efcebed6637079dfd826728fcbdfd856540c3d04f4R433 Now it should be in streamEvent after check. Since now we will output all events

Reply:
well this it is in streamEvent.